src/command_modules/azure-cli-network/azure/cli/command_modules/network/_params.py

```diff
@@ -238,6 +241,11 @@ def completer(prefix, action, parsed_args, **kwargs): # pylint: disable=unused-a
 
 # VPN gateway
```

Comment:
># VPN gatewa [](start = 0, length = 12)

It won't let me put comments in the description, to annotate the help text. Anyways, the text for tags ends with "Use "" for None". However, since this is a create, and tags are optional, you should simply omit the --tags option for None. Use "" for None is only really relevant when updating tags.  I suspect this occurs in more places.